prow/plugins.yaml

```diff
@@ -67,7 +71,7 @@ approve:
   - kubernetes/steering
   - kubernetes/utils
   - kubernetes-incubator/ip-masq-agent
-  implicit_self_approve: true
+  require_self_approval: false
```

Comment:
TBH I find this name more confusing because it could be read as "the PR author must approve their own change"

Reply:
I agree the option name is confusing, though I have difficulty coming up with another one that makes `false` mean the desired behaviour without a double negative.

(Perhaps we could instead just keep `implicit_self_approve` and make it default true?)

Reply:
No

Reply:
`explicit_self_approval`?

Reply:
I like explicit_self_approval, though I still like the original too... it is not clear to me that we have a blanket stance of "all our config settings must be worded such that their default behavior is `true` and overriding the behavior is `false`"

Reply:
(I would expect us to code such that "best" / default behavior is false since that's the way golang structs would intialize, but that's maybe an implementation-detail reason for it)

Reply:
explicit_self_approval sounds better to me. I want to choose something that has a default of false